index.js

```diff
@@ -0,0 +1,2 @@
+var esVersion = require('./version').get();
+module.exports = require('./' + esVersion);
```

Comment:
If we go with using a separate library, can we try-catch this and load what the package.json points to if its missing?
Thinking that this could be useful in case let's say you update pinejs-client-js and start using the newly released es3000 build output, while you still are using balena-xyz which has only support for es2015.